texi2rst.py

```diff
@@ -1677,6 +1683,13 @@ def test_option_ref_with_var(self):
         out = self.make_rst_string(doc)
         self.assertEqual('This is similar to how :option:`-Walloca-larger-than` = byte-size works.\n\n', out)
 
+    def test_option_ref_with_var2(self):
+        xml_src = '<para>See also <option>-Walloca-larger-than=<var>byte-size</var></option>.</para>'
+        doc = from_xml_string(xml_src)
+        doc = fixup_option_refs(doc)
+        out = self.make_rst_string(doc)
+        self.assertEqual('See also :option:`-Walloca-larger-than` = byte-size.\n\n', out)
```

Comment:
Similar here; would be better for the `byte-size` to have some .rst markup

Reply:
Yes, will do a separate pull request once I'm happy with the generated `rst` files.